pkg/fqdn/name_manager.go

```diff
@@ -8,6 +8,7 @@ import (
 	"net"
 	"net/netip"
 	"regexp"
+	"slices"
```

Comment:
We need to replace this by `golang.org/x/exp/slices` in `v1.14` for Go 1.20 which doesn't have the standard library `slices` package yet.